.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- 
+---------------------------------------------------
+GENERAL SUPPORT INFORMATION
+---------------------------------------------------
+
+The GitHub issue tracker is for bug reports and feature requests.
+General support can be found at the following locations:
+
+Example configs:
+
+3.0 - https://github.com/inspircd/inspircd/tree/master/docs/conf
```

Comment:
Maybe put 2.0 before 3.0 for now and put "3.0 (alpha)" to make it clear that its an alpha release.